comiccrawler/core/__init__.py

```diff
@@ -672,3 +680,10 @@ def analyze_info(mission, mod):
 	remove_duplicate_episode(mission)
 
 	print("Analyzing success!")
+
+def format_number(title, format):
+	"""第3卷 --> 第003卷"""
+	def replacer(match):
+		number = match.group()
+		return format.format(int(number))
+	return re.sub("\d+", replacer, title)
```

Comment:
![http://i.imgur.com/LzP609B.png](http://i.imgur.com/LzP609B.png)
I didn't see the problem.